apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveRoutes.kt

```diff
@@ -260,6 +268,7 @@ internal fun Route.oppgaveRoutes(service: OppgaveService) {
                 inTransaction {
                     val oppgaver = service.hentFristGaarUt(request)
                     oppgaver.forEach {
+                        service.fjernSaksbehandler(it.oppgaveID)
```

Comment:
Nå vil vel dette skje for alle automatiske? Men vi ønsker det vel kun for de som fyller 20?

Reply:
Ja, tenkte på det sjølv no nettopp, at _vent litt, det her skurrar__, så oppdaterte no.